README.md

```diff
@@ -0,0 +1,25 @@
+# Loop Development Kit
+
+This is the central repository for all the Olive Help LDKs. You'll probably want to look at the individual LDK readmes in the `ldk\{language}` folder.
+
+## Developing
+
+### LDK Contract
+
+Each LDK must adhere to this contract:
+
+- They should start a GRPC server serving the following services:
+    - `proto.Loop`
+    - `plugin.Broker`
+    - `plugin.Stdio`
+- When started up they write to STDOUT a connection string in the following format: `1|1|tcp|{HOST}:{PORT}|grpc`.
+- They do not write anything else to STDOUT until the Go LDK LoopClient calls the `proto.Loop.LoopStart` rpc.
```

Comment:
This will never stop being complicated :D 
In this particular scenario it's the Go LDK's `LoopClient` that's consumed by the Olive Helps app to connect to the `LoopServer` provided in each of the LDKs.

Reply:
Ohhhh, ok. Thanks for clarifying it to me!